ideas/282-deterministic-builds.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 282-deterministic-builds
+title: Deterministic Mobile App Builds
+status: In Progress
+created: 2018-08-16
+category: infra
+lead-contributor: jakubgs
+contributors:
+exit-criteria: yes
+success-metrics: yes
+clear-roles: yes
+future-iterations: yes
+roles-needed:
```

Comment:
Seems like it'd be useful to have Clojure/Go devs, or not?

Reply:
And perhaps security @corpetty as well, not sure

Reply:
Yes, definiely.